src/Resources/contao/widgets/TextArea.php

```diff
@@ -101,6 +107,39 @@ public function __set($strKey, $varValue)
 
 
 	/**
+	 * {@inheritdoc}
+	 */
+	public function getDcaFilter()
```

Comment:
The filters depend on the table of the DCA. Why can I not set a root array for the `tl_page` here?